### ISSUE TITLE
epiuser does not have the correct permissions to run epicli and access the /shared folder.

### ISSUE THREAD

**Issue by martamercury** [epiphany-platform/epiphany#1436]
We were not able to run epicli as epiuser due to permission errors during install. We had to sudo to root.

**To Reproduce**
From the DE 1.3 On Premise Install guide...

1-On the Installer Machine, execute the following commands:
```
# docker run -it -v <StagingDirectory>:/shared \
-e http_proxy=<HTTP_PROXY> \
-e https_proxy=<HTTPS_PROXY> \
-e HTTP_PROXY=<HTTP_PROXY> \
-e HTTPS_PROXY=<HTTPS_PROXY> \
--rm epiphanyplatform/epicli:0.6.0
```
Where:
```
  <StagingDirectory> is the path to the staging directory you created in step 1.
  <HTTP_PROXY> is the hostname or IP address of your network's HTTP proxy, if applicable. Exclude these lines if no HTTP proxy is applicable.
  <HTTPS_PROXY> is the hostname or IP address of your network's HTTPS proxy, if applicable. Exclude these lines if no HTTPS proxy is applicable
```
A bash shell will start.

2-In the bash shell, execute the following command:
bash-5.0# epicli apply -f singlemachine.yml

**Expected behavior**
you should have the right permissions as the epiuser

**Config files**
If applicable, add config files to help explain your problem.

**OS (please complete the following information):**
bash-5.0# uname -a
Linux 997677745ffb 4.15.0-96-generic #97-Ubuntu SMP Wed Apr 1 03:25:46 UTC 2020 x86_64 GNU/Linux

**Cloud Environment (please complete the following information):**
on premise

**Additional context**

**Comment by mkyc** [comment]
I removed it from 0.6.1 **milestone** and added it to 0.6.1 **release**.

**Comment by DaOsie** [comment]
@martamercury I wasn't able to reproduce your problem, therefore I would need more information about:
- docker and docker-engine version
- file owners and its permissions after starting epicli container
From the kernel version, I assume that your on prem machine was running ubuntu 18.04?

**Comment by mkyc** [comment]
@przemyslavic or @to-bar can you guys say if you have any problem of this kind even on 0.6.x clusters?

**Comment by przemyslavic** [comment]
@mkyc it looks like this was fixed in 0.7 [#1378](https://github.com/epiphany-platform/epiphany/pull/1378)

**Comment by mkyc** [comment]
Yeah it definitely looks like. I'm closing this one.
